source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,126 @@
+
+* Copying a configuration to a host with the same interface names but different physical network connections will
+produce a bad configuration. Only copy configurations to hosts that have the same interface names and physical 
+network connections. 
+
+* The destination host has to have an equal or greater number of interfaces as the original host. Otherwise,
+the feature cannot determine what to do.
+
+* The `ovirtmgmt` network stays on the original interface which is excluded from the operation. 
+
+
+### Implementation
+
+The engine has a new method, CopyNetworks, that copies the network configuration of the source host (SourceHost)
+and uses it to overwrite the network configuration of the destination host (DestinationHost).
+
+This method takes two parameters, the IDs of the SourceHost and DestinationHost.
+
+Given the parameters, the method validates that the following items are correct:
+
+* Both hosts are in the same cluster.
+* The destination host has an equal or higher number of interfaces than the source host.
+
+(NOTE: These interface names and physical connections to the datacenter must be identical. Otherwise, this sorting
+process produces a bad network configuration on the destination host.)
+
+The method prepares a set of network attachment definitions for the DestinationHost by doing the following:
+
+* Lists the interfaces on the SourceHost that don't have bonds.
+* Lists all of the interfaces on the DestinationHost.
+* Sorts both lists by name.
+* Excludes interface/bonds connected to the ovirtmgmt network.
+* Pairs all of the bond slaves from the SourceHost with their corresponding partners on the DestinationHost.
+* Creates bond definitions in a configuration for the DestinationHost.
+* Retrieves a list of all interfaces with bonds from SourceHost.
+* Excludes bond slaves from both lists.
+* Shortens the DestinationHost list to match the same length as SourceHost list.
+* Copies the network attachments for all interfaces in SourceHost list.
+* Walks through the new network attachments list and updates any static IP address configurations.
+* Sends HostSetupNetworks with new attachments and bonds for DestinationHost to complete everything in a single
+ transaction.
+* If there is an error, it reverts the configuration changes.
+
+The Admin Portal has a new button called Copy Host Networking. Clicking the button opens a dialog window. There, 
+you can select the source host from which to copy the configuration.
+
+
+### Future work
+
+To improve usability the biggest limitation is network mapping of interfaces with different name. 
+This can be overcome by enabling custom mapping which would let the administrator to map certain
+or all interfaces to their corresponding equivalents on the destination host. This way the limitation
+of interface count on destination host could be removed assuming that the administrator would provide
+specific mapping for this setups. This functionality could be enabled via UI and REST API.
+
```

Comment:
We could label nic configurations, and apply them to nics on new hosts by just labelling them.
This would even allow to copy configuration from two different hosts onto one.
For example:

HOST_SRC0                 HOST_SRC1                 HOST DEST
nic0 (label0)  ------------------------------------------->   nic0 (label0)  
nic 1 (label1) ------------------------------------------->   nic 1 (label1)
nic3                               nic0 (label2)   ----------->    nic2 (label2) 
                                      nic1  (label3)  ---------->    nic3 (label3)
                                                                                nic4

Reply:
Label is possible solution but I am afraid that would create mess with labels.

Reply:
I agree with Ales here..prefer not to